.env.example

```diff
@@ -65,3 +65,5 @@ ELASTIC_MIGRATIONS_DIRECTORY=elastic/migrations
 SCOUT_DRIVER=elastic
 
 DEFAULT_THUMB_URL=
+
+CURRIKI_DEMO_EMAIL=demo@currikistudio.org
```

Comment:
Add GAPI_CLIENT_ID env to .env.example too

Reply:
It is already there.